PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -90,18 +103,88 @@ local SetCameraOnSpawn = true
 
 local hasGameLoaded = false
 
+local GestureArea = nil
+
+do
+	-- Check for changes in ViewportSize to decide if PortraitMode
+	local function onWorkspaceChanged(property)
+		if property == 'CurrentCamera' then
+			local newCamera = workspace.CurrentCamera
+			if newCamera then
+				local size = newCamera.ViewportSize
+				PortraitMode = size.X < size.Y
```

Comment:
Seems like a toss up. At any rate, soon we'll want to switch to relying on Kyle's ScreenOrientation event.